Anytype/Sources/PresentationLayer/Modules/Chat/Subviews/Message/Reaction/MessageReactionView.swift

```diff
@@ -8,6 +8,9 @@ struct MessageReactionView: View {
     let onTap: () async throws -> Void
     let onLongTap: () -> Void
     
+    @Environment(\.messageYourBackgroundColor)
+    private var messageYourBackgroundColor
```

Comment:
I suggest to clean up the naming a bit, "your" is a bit misleading atm

messageYourBackgroundColor -> userSelectedMessageBackgroundColor / CustomMessageBackgroundColor etc.